### DISCUSSION TITLE
documentation or videos on how to use

### DISCUSSION THREAD

**Discussion by Kreijstal** [OpenAdaptAI/OpenAdapt#817] [General]
Maybe this will become easier to understand for general audience or for other devs as well, right now the readme explains what it is but contains no examples of usage

**Comment by abrichr** [comment]
Thanks @Kreijstal ! We are working on updating our documentation in this PR: https://github.com/OpenAdaptAI/OpenAdapt/pull/771#issuecomment-2198573742

Edit: you can see the preview here: https://openadapt.gitbook.io/openadapt.ai/~/revisions/jCu5FSn9t0uOA3QTp1AQ/getting-started/readme/user-guide
